packages/core/src/interfaces/axis-scales.ts

```diff
@@ -32,6 +33,10 @@ export interface AxisOptions {
 	 * optional title for the scales
 	 */
 	title?: string;
+	/**
+	 * thresholds
```

Comment:
We probably should include some more details on how the thresholds should be used. Maybe the documentation is clear enough just by reading the typings, but an example would probably help?